scripts/docker/gatkbase/install_R_packages.R

```diff
@@ -6,34 +6,27 @@
 # pull request tests in order to make sure it's still working                 #
 ###############################################################################
 
-#Make sure to use http not https as this will give an "unsupported URL scheme" error
-getoptUrl="http://cran.r-project.org/src/contrib/Archive/getopt/getopt_1.20.0.tar.gz"
-if (!("getopt" %in% rownames(installed.packages()))) {
-  install.packages(getoptUrl, repos=NULL, type="source", clean = TRUE)
-}
-optparseUrl="http://cran.r-project.org/src/contrib/Archive/optparse/optparse_1.3.2.tar.gz"
-if (!("optparse" %in% rownames(installed.packages()))) {
-  install.packages(optparseUrl, repos=NULL, type="source", clean = TRUE)
-}
-datatableUrl="http://cran.r-project.org/src/contrib/Archive/data.table/data.table_1.10.4-2.tar.gz"
-if (!("data.table" %in% rownames(installed.packages()))) {
-    install.packages(datatableUrl, repos=NULL, type="source", clean = TRUE)
-}
-dependencies = c("ggplot2","gplots","gsalib")
-repos <- c("http://cran.cnr.Berkeley.edu",
-           "https://cran.mtu.edu",
-           "http://lib.stat.cmu.edu/R/CRAN/")
-missing <- which(!(dependencies %in% rownames(installed.packages())))
-try <- 1
-while(length(missing)!=0 & try <= length(repos)) {
-    install.packages(dependencies[missing], repos = repos[try], clean = TRUE)
-    missing <- which(!(dependencies %in% rownames(installed.packages())))
-    try <- try + 1
+options(warn = 2)     # treat warnings as errors, otherwise script can fail silently if a package fails to install
+
+InstallPackageFromArchive = function(packageName, packageURL) {
+    # make sure to use http not https as this will give an "unsupported URL scheme" error
+    if (!(packageName %in% rownames(installed.packages()))) {
+        install.packages(packageURL, repos = NULL, type = "source", clean = TRUE)
+    }
 }
 
+dependencies = c("gplots",
+                 "digest", "gtable", "MASS", "plyr", "reshape2", "scales", "tibble", "lazyeval")    # ggplot2
```

Comment:
Just realized the comment might not be very helpful.  I'll change it from "ggplot2" to "for ggplot2".